src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1233,7 +1233,21 @@ public void validateAlternateAlleles() {
             throw new TribbleException.InternalCodecException(String.format("one or more of the ALT allele(s) for the record at position %s:%d are not observed at all in the sample genotypes", getContig(), getStart()));
     }
 
+    private void validateNumberOfItems(final String attributeKey, final int observedSize ) {
```

Comment:
The rest of the code was using statistics lingo, `observed/reported` but will change to `actual/expected`.